user-console/src/main/java/org/pentaho/mantle/client/MantleApplication.java

```diff
@@ -495,11 +492,14 @@ private void loadWhitelistedHosts() {
       requestBuilder.sendRequest( null, new RequestCallback() {
         @Override public void onResponseReceived( Request request, Response response ) {
           List<String> whitelistedHosts = new ArrayList<>();
-          JSONArray data = new JSONArray( JsonUtils.safeEval( response.getText() ) );
-          for ( int i = 0; i < data.size(); i++ ) {
+
```

Comment:
You'll need test for MantleApplication as well. You can use this as an reference https://github.com/pentaho/pentaho-platform/blob/master/extensions/src/test/java/org/pentaho/platform/web/http/api/resources/EmailResourceTest.java

You won't have to do the MicroPlatform logic in #setup 

- https://github.com/pentaho/pentaho-platform/blob/master/extensions/src/test/java/org/pentaho/platform/web/http/api/resources/EmailResourceTest.java#L50-L52

Also, you won;t have to do the KettleEnvironment logic either

- https://github.com/pentaho/pentaho-platform/blob/master/extensions/src/test/java/org/pentaho/platform/web/http/api/resources/EmailResourceTest.java#L59
- https://github.com/pentaho/pentaho-platform/blob/master/extensions/src/test/java/org/pentaho/platform/web/http/api/resources/EmailResourceTest.java#L92